src/rpdk/core/contract/suite/contract_asserts.py

```diff
@@ -106,11 +110,12 @@ def response_contains_primary_identifier(resource_client, response):
 def response_contains_unchanged_primary_identifier(
     resource_client, response, current_resource_model
 ):
-    resource_client.is_primary_identifier_equal(
+    assert resource_client.is_primary_identifier_equal(
```

Comment:
is_primary_identifier_equal is already asserting do we need to assert here too?

Reply:
[is_primary_identifier_equal](https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/resource_client.py#L389) has expected `boolean` output; it does not assert